packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -48,6 +57,12 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
         address gasPayingToken;
     }
 
+    struct Roles {
+        address owner;
+        address feeAdmin;
+    }
+    // TODO: add unsafe block signer and batcher hash?
```

Comment:
It could make sense to do this, we could also do batcher as address rather than bytes32 to improve the ux. Also could be good to rename owner to chainOperator or chainGovernor